src/scss/_leftbar.scss

```diff
@@ -136,71 +105,8 @@ div.navbar-brand-box {
 }
 
 nav.bd-links {
-    margin-left: 0px;
```

Comment:
You will need to keep this line I think. Apparently in PST we have `margin-left: -15px;`, which makes that in the preview (https://sphinx-book-theme--324.org.readthedocs.build/en/324/) the sidebar navigation is shifted to the left